### ISSUE TITLE
Sending packet to CubeSates

### ISSUE THREAD

**Issue by hooman-sarv** [G4lile0/tinyGS#184]
Hello. How can we send a package to CubeSats? Can we only receive packages on this platform? If we want to send and receive our packages, how can we?

**Comment by thefatmoop** [comment]
I am wondering the same thing - it's cool to be a free base station for some project, but can we send data up to the sats and get it off an api?

**Comment by G4lile0** [comment]
Yep, it's possible, we already had some satellites with relay capacity, pitifully not active now.
We may have more satellite in the future, but to be able to transmit you have to check your local legislation and own a ham license.
